plugins/basename/help.md

```diff
@@ -1,10 +1,10 @@
 # Description
 
-This plugin is used to get the last item of a file path or URL using Python's basename utility
+This plugin is used to get the last item of a file path or URL using Python's Basename utility
 
 # Key Features
   
-* Obtain basename of a file path or URL.
+* Obtain Basename of a file path or URL.
```

Comment:
we should keep this at basename as it's not a proper noun.

Reply:
When it's changed to lower case it throws an error with the helpValidator, it mentions that  the help section `contains non-matching title`. Should I keep it as lowercase and link the ticket that raises this error?

Reply:
We should update the validator - it shouldn't hold up this ticket however.